src/get-started/install.md

```diff
@@ -44,9 +44,22 @@ For distributives with `deb`-based package managers (such as Debian or Ubuntu),
 use
 
 ```shell
-apt-get install libssl-dev pkg-config
+apt-get install libsodium-dev libsnappy-dev \
+    librocksdb-dev libssl-dev pkg-config
 ```
 
+`libsodium-dev`, `libsnappy-dev` and `librocksdb-dev` are optional libraries
+for installation. Meanwhile, as these packages reduce the build time, their
+installation is highly recommended.
+
+You should also set the paths to libraries of these packages in the
+environment variables:
+
+```shell
+export ROCKSDB_LIB_DIR=/usr/lib/x86_64-linux-gnu
```

Comment:
Rocksdb seems to be in /usr/lib:
```
$ find /usr/lib/ -iname *rocks*
/usr/lib/librocksdb.so.5
/usr/lib/librocksdb.so.5.17.2
/usr/lib/librocksdb.so.5.17
/usr/lib/librocksdb.so
```

https://github.com/exonum/exonum-java-binding/blob/35fb3b5c2fc4372ec110602f52ef088e612ba805/.travis.yml#L34-L35

Reply:
That's on Ubuntu Linux, Mac might have different paths

Reply:
It depends on package config as well.

Reply:
BTW, we no longer offer/maintain our PPA, do we?

Reply:
For fresh distributives it's unnecessary, I suppose.